backstop/tests/iModel-loader.html

```diff
@@ -0,0 +1,398 @@
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.05"
+              />
+            </polygon>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-4"
+              points="184.99 107.14 277.48 160.54 369.98 213.94 277.49 267.34 185 320.75 184.99 320.75 184.99 107.14"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.22"
+              />
+            </polygon>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-5"
+              points="184.99 107.14 184.99 320.74 0 213.94 184.99 107.14"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.28"
+              />
+            </polygon>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-2"
+              points="369.93 0 554.92 106.81 462.43 160.21 369.98 213.58 369.98 0.33 369.93 0.36 369.93 0"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.11"
+              />
+            </polygon>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-3"
+              points="369.98 0.33 369.98 213.94 277.48 160.54 184.99 107.14 369.93 0.36 369.98 0.33"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.17"
+              />
+            </polygon>
+            <path
+              class="iui-loading-imodel-i"
+              d="M369.82,133.26a59.58,59.58,0,1,1-59.58,59.57A59.58,59.58,0,0,1,369.82,133.26ZM538,648.53a14,14,0,0,1-6.84,12l-152.8,91a14,14,0,0,1-14.29,0L208.5,659.71a14,14,0,0,1-6.89-12.07V566.85a14,14,0,0,1,14-14,13.84,13.84,0,0,1,7.16,2l91,50.38V412.85l-49.22-31.36a14,14,0,0,1-6.85-12.05V288.37a14,14,0,0,1,21.21-12l140.18,77.56a14,14,0,0,1,6.82,12V605.19l91-50.38a13.84,13.84,0,0,1,7.16-2,14,14,0,0,1,14,14Z"
+            />
+          </g>
+        </svg>
+
+        <h1 class="iui-text-headline">Sample loading</h1>
+        <p class="iui-text-subheading iui-text-spacing">This is sample use case.</p>
+        <p
+          class="iui-text-subheading iui-text-spacing"
+          style="display: none;"
+        >You can find this loader in iTwinUI illustrations package.</p>
```

Comment:
I pulled your branch and this text is not being centered like it is in the pr description. Is it still working for you?
![image](https://user-images.githubusercontent.com/88331924/146201591-ac26c73b-e91a-4884-bce2-b7e9f0dd5e8d.png)

Reply:
Yeah.. I think some classNames were wrong. Fixed it now.